api/tests/certification/configuration/integration/infrastructure/repositories/active-calibrated-challenge-repository_test.js

```diff
@@ -0,0 +1,134 @@
+        challengeId: 'rec4567',
+        calibrationId: calibration.id,
+      });
+      const firstActiveCalibratedChallenge = datamartBuilder.factory.buildActiveCalibratedChallenge({
+        challengeId: 'rec1234',
+        calibrationId: calibration.id,
+      });
+      // from CLEA scope
+      datamartBuilder.factory.buildActiveCalibratedChallenge({
+        challengeId: 'rec1234',
+        calibrationId: cleaCalibration.id,
+      });
+      // with other calibrationId
+      datamartBuilder.factory.buildActiveCalibratedChallenge({
+        challengeId: 'rec4567',
+        calibrationId: otherCalibration.id,
+      });
+      // with calibration to validate
+      datamartBuilder.factory.buildActiveCalibratedChallenge({
+        challengeId: 'rec6789',
+        calibrationId: toValidateCalibration.id,
+      });
+
+      const expectedActiveCalibratedChallenges = [
+        domainBuilder.certification.configuration.buildActiveCalibratedChallenge({
+          scope: firstActiveCalibratedChallenge.scope,
+          discriminant: firstActiveCalibratedChallenge.alpha,
+          difficulty: firstActiveCalibratedChallenge.delta,
+          challengeId: firstActiveCalibratedChallenge.challenge_id,
+        }),
+        domainBuilder.certification.configuration.buildActiveCalibratedChallenge({
+          scope: secondActiveCalibratedChallenge.scope,
+          discriminant: secondActiveCalibratedChallenge.alpha,
+          difficulty: secondActiveCalibratedChallenge.delta,
+          challengeId: secondActiveCalibratedChallenge.challenge_id,
+        }),
+      ];
+
+      await datamartBuilder.commit();
+
+      //when
+      const calibratedChallenges = await activeCalibratedChallengeRepository.findByComplementaryKeyAndCalibrationId({
+        complementaryCertificationKey: ComplementaryCertificationKeys.PIX_PLUS_DROIT,
+        calibrationId: calibration.id,
+      });
+
+      //then
+      expect(calibratedChallenges).to.deep.equal(expectedActiveCalibratedChallenges);
+    });
+
+    it("should return empty array when calibration id given doesn't match the given scope", async function () {
+      const calibration = datamartBuilder.factory.buildCalibration({
+        scope: ComplementaryCertificationKeys.PIX_PLUS_PRO_SANTE,
+        status: 'VALIDATED',
+      });
+
+      datamartBuilder.factory.buildActiveCalibratedChallenge({
+        challengeId: 'rec1234',
+        calibrationId: calibration.id,
+      });
+
+      await datamartBuilder.commit();
+
+      //when
+      const calibratedChallenges = await activeCalibratedChallengeRepository.findByComplementaryKeyAndCalibrationId({
+        complementaryCertificationKey: ComplementaryCertificationKeys.PIX_PLUS_DROIT,
+        calibrationId: calibration.id,
+      });
+
+      //then
+      expect(calibratedChallenges).to.deep.equal([]);
+    });
+
+    it('should return empty array when calibration is to validate', async function () {
```

Comment:
The test est faux, il n'y a actuellement pas de filtre sur le status de `data_calibrations` dans le repository.
Mais je suis d'accord de garder le test, il faut ajouter le status dans le where clause du repo.